tests/cypress/e2e/actions_users/registration_involved/case_28_review_pipeline_feature.js

```diff
@@ -291,6 +291,22 @@ context('Review pipeline feature', () => {
             cy.get('.cvat-issues-sidebar-previous-frame').should('be.visible').click();
             cy.checkFrameNum(1);
 
+            const framesWithIssues = [[0, 1, 'Demo 1'], [1, 1, customIssueDescription], [2, 1, customIssueDescription]];
```

Comment:
We already have `countIssuesByFrame` for such purpose, why we need to add one more?